apps/home/app/@readme/people/page.tsx

```diff
@@ -0,0 +1,25 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import { Mdx } from '@/_shared'
+import { ReadmeBox } from '../_shared'
+
+export default async function Page({
+  searchParams,
+}: {
+  searchParams: { username: string }
+}) {
+  const { username } = searchParams
+
+  try {
+    const filePath = path.join(process.cwd(), 'app/@readme/_shared/content', `${username}.mdx`)
+    const readme = fs.readFileSync(filePath, 'utf8')
+
+    return (
+      <ReadmeBox>
+        <Mdx fileText={readme} />
+      </ReadmeBox>
+    )
+  } catch (e) {
+    return <ReadmeBox>클라이머로 등록되지 않는 멤버예요! 🧐</ReadmeBox>
+  }
```

Comment:
```suggestion
  } catch {
    return <ReadmeBox>클라이머로 등록되지 않는 멤버예요! 🧐</ReadmeBox>
  }
```

e 객체 안 쓸거면 안 써도 된다구 하더라구요 ! 땡스투 제로초!

Reply:
[4536c91](https://github.com/hamsurang/origin/pull/32/commits/4536c915eb733aef73accad7f69fcde07d2f8db3)에서 미들웨어 처리로 인해 사용되지 않아 제거하였어요! 😆